src/Data/Event/EventEditingConfig.ts

```diff
@@ -0,0 +1,24 @@
+import { provideEditingConfig } from 'scrivito'
+import { Event } from './EventDataClass'
+
+provideEditingConfig(Event, {
+  title: 'Event',
+  attributes: {
+    keyword: { title: 'Keyword' },
```

Comment:
* [ ] FYI AFAICS this is used as "Title" not "Keyword"?

Reply:
I tried to use the same attribute names as PisaSales. E.g. Quote and Order also have a `keyword: { title: 'Keyword' },`.

Reply:
I don't know enough about the backend. As long as the "Keyword" is not used as a "Title" in the content, fine with me.